jenkins-scripts/docker/lib/debbuild-base.bash

```diff
@@ -200,7 +207,7 @@ while (! \$update_done); do
 done
 
 # new versions of mk-build-deps > 2.21.1 left buildinfo and changes files in the code
-rm -f ${PACKAGE_ALIAS}-build-deps_*.{buildinfo,changes}
+rm -f \${PACKAGE_ALIAS}-build-deps_*.{buildinfo,changes}
```

Comment:
I don't understand the change on this line

Reply:
All this code goes into a generated build.sh. There are two rounds for that script:
 1. Generation of the script: variable expanded in this first step are the ones coming from out of the own script, mainly input parameters of the Jenkins job.
 2. The script is being called with the variables without escaping being resolved at this point. If we leave `PACKAGE_ALIAS` here without escaping, the changes in this PR done to `PACKAGE_ALIAS` are not taking into account since it was already resolved. So we need to escape it in order to get the new value added in this PR some lines above.

Clarified or the doubt was different?

Reply:
thank you! some of the variable are not escaped (like line 218), which is why I asked